TwoWayDatabinding/README.md

```diff
@@ -0,0 +1,20 @@
+# Two-Way Databinding
+This sample is designed to demonstrate two-way databinding. It's a simple inventory app, with the theme Toy Inventory. 
+It contains a single activity with two fragments: First fragment shows the list of toys, whereas second fragment is used for adding or editing toys.
+You can see the use of two-way databinding in AddToyFragment. We used edittext, checkboxes, radiobuttons and a spinner that all works two-way:
+It shows the chosen toy when you're editting a toy, but it also listens for changes. When you are entering a new entry, it is again listening for the changes simultaneously and keeps it in viewmodel. And all these without setting any listeners from java code.If you have previously done an inventory app without databinding, you'd be surprised how this is done with very few code. 
```

Comment:
@OyaCanli the README file looks all nice but is there another way to write it in even more simpler way? I mean, in a way where we can explain the working way of two-way-databinding as a brief but still the actual working than just giving the idea? Or should we leave that part for tutorial?